services/placement/service/placementservice_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/m3db/m3cluster/services/placement"
 	"github.com/m3db/m3cluster/shard"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
`require` is good for if you want the test to bailout without running the rest of the checks.  There are some scenarios where that's more desirable (i.e. would cause a nil panic later in the test, hence better to use a require above it to ensure that doesn't happen). 

Assert is more preferable if it won't crash however as you get more signal back about what things are failing and where when tests start to fail.